src/public/modules/forms/services/mailto.client.factory.js

```diff
@@ -0,0 +1,39 @@
+const dedent = require('dedent-js')
+
+angular.module('forms').factory('MailTo', ['$window', MailTo])
+
+function MailTo($window) {
+  /**
+   * Generates a mailto URI which opens the client's default email application
+   * with an email to share their secret key. This requires the form title and
+   * secret key at minimum, and optionally the form ID.
+   * @param {string} title
+   * @param {string} secretKey
+   * @param {string} [formId]
+   */
+  const generateMailToUri = (title, secretKey, formId) => {
+    // Construct body piece by piece to avoid confusion with order of operations
+    let body = dedent`
+      Dear collaborator,
+
+      I am sharing my form's secret key with you for safekeeping and backup. This is an important key that is needed to access all form responses.
+
+      Form title: ${title}`
+    if (formId) {
+      body += `Form URL: https://form.gov.sg/${formId}`
```

Comment:
we should not be hard coding in `https://form.gov.sg`. Please use the defaults available?

Reply:
agree, you mean `LINKS.APP.ROOT` in the translations right?

Reply:
that resolves to `https://form.gov.sg` anyway, but it won't be a hardcode

Reply:
`window.appUrl` i believe

Reply:
lets use the translations file instead. That seems to be what is consistently used by the frontend. so `LINKS.APP.ROOT`